packages/amplify-util-mock/src/api/api.ts

```diff
@@ -80,7 +81,7 @@ export class APITest {
     let config: any = processAppSyncResources(stack, transformerOutput);
     await this.ensureDDBTables(config);
     this.transformerResult = this.configureDDBDataSource(config);
-    this.transformerResult = this.configureLambdaDataSource(config);
+    this.transformerResult = this.configureLambdaDataSource(context, config);
```

Comment:
line 83 also assign to ```this.transformerResult```, is it intentional?

Reply:
It works, but I think it is mutating the same is being assigned multiple times. In a follow up PR I am going to clean this up and make input config in helper function immuteable